.github/workflows/stack.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   build:
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-18.04]
```

Comment:
I'm all for specifying exact versions, but why are we downgrading from 20.04?

Reply:
It fixes the `stack` build. 😁

Reply:
I would prefer if it just worked on the current ubuntu release. Downgrading to something so old could cause other headaches in future.

Reply:
True.  I'll remove it.